manifests/init.pp

```diff
@@ -3,6 +3,7 @@
   Boolean $manage_database_maintenance = true,
   Boolean $manage_postgresql_settings  = true,
   Boolean $manage_table_settings       = true,
+  Integer $resource_events_ttl         = 0,
```

Comment:
Either the resource_events_ttl class should be included here in init.pp and not in maintenance... OR this parameter shouldn't be here and should only be on maintenance

Reply:
Ok